src/test/java/com/google/step/snippet/external/GeeksForGeeksClientTest.java

```diff
@@ -11,7 +11,12 @@
 @RunWith(JUnit4.class)
 public final class GeeksForGeeksClientTest {
 
-  private final GeeksForGeeksClient client = new GeeksForGeeksClient("CSE_ID");
+  private final GeeksForGeeksClient client =
+      new GeeksForGeeksClient("CSE_ID") {
+        public String getVotes(String url) {
```

Comment:
Why public? But why is this necessary at all? This is the same implementation as in Client.java.

Reply:
It won't be the same as the implementation in Client.java in the next PR.

Reply:
Also, this cant be public because the override cant reduce the visibility of the method inside Client.java

Reply:
Can this be done in the next PR instead of this one??